app/views/home/_chapter_program_manager.html.erb

```diff
@@ -0,0 +1,68 @@
+<% #Add the 'display-open' h3 class to display as opened on load %>
+<h3 class="display-open"><span class="chevron-closed">&nbsp;</span>Chapter Program Manager</h3>
+<div>
+	<h5>Location</h5>
+	<p>Remote</p>
+	<h5>About Girl Develop It</h5>
+	<p>
+		Girl Develop It is a 501(c)3 nonprofit organization that provides affordable and judgment-free opportunities for adult women interested in learning web and software development. Through in-person classes and community support, Girl Develop It helps women of diverse backgrounds achieve their technology goals and build confidence in their careers and their everyday lives.
+	</p>
+	
+	<h5>About This Role</h5>
+	<p>Girl Develop it is seeking an enthusiastic, experienced Program Manager to manage all aspects of our national chapter-based program and to spearhead a robust expansion strategy. The Chapter Program Manager will be responsible for supporting current chapters to ensure that program model is being implemented consistently with GDI’s Theory of Change; to provide technical assistance to chapter leadership team; and to execute a focused expansion strategy aimed at revitalizing stagnant chapters and growing new chapters.</p>
+	
+	<h5>Duties &amp; Responsibilities</h5>
+	<p>
+		The <bold>Chapter Program Manager</bold> role will report directly to the Program Director and work very closely with members of senior management and chapter leadership teams to provide crucial support to our 55 chapters in serving our 84,000+ members, as well as expanding our reach to new markets.
+	</p>
+		
+	<p>
+		As a core member of the national leadership (HQ) team, the Chapter Program Manager will:
+	</p>
+	<ul>
+		<li>Collaborate with national and Chapter Leadership teams, consultants and key partners to increase women’s access to affordable, in-person web and software development classes, nationwide as outlined in GDI’s mission and vision</li>
```

Comment:
shouldn't pose an issue, but might help to keep an eye on the copy/pasted special characters e.g. apostrophes since they might not transfer very well to various browsers 👍